components/dm/spec/bindversion.go

```diff
@@ -0,0 +1,17 @@
+package spec
+
+import "github.com/pingcap/tiup/pkg/cluster/spec"
+
+// DMComponentVersion maps the dm version to the third components binding version
+func DMComponentVersion(comp, version string) string {
+	switch comp {
+	case spec.ComponentAlertManager:
+		return "v0.17.0"
+	case spec.ComponentGrafana:
+		return "v4.0.3"
```

Comment:
Grafana and  prometheus will be released with TiDB, their version should not be hard coded.

Reply:
This is similar bind the version with a specified version of grafana&prometheus, but we use the one tidb used.
Every version of DM will using this version of grafana and prometheus.